waku/v2/protocol/waku_rln_relay/waku_rln_relay_utils.nim

```diff
@@ -809,10 +809,11 @@ proc validateMessage*(rlnPeer: WakuRLNRelay, msg: WakuMessage,
     waku_rln_invalid_messages_total.inc(labelValues=["invalid_epoch"])
     return MessageValidationResult.Invalid
 
-  if not rlnPeer.validateRoot(msg.proof.merkleRoot):
-    debug "invalid message: provided root does not belong to acceptable window of roots", provided=msg.proof.merkleRoot, validRoots=rlnPeer.validMerkleRoots
-    waku_rln_invalid_messages_total.inc(labelValues=["invalid_root"])
-    return MessageValidationResult.Invalid
+  ## TODO: FIXME
+  # if not rlnPeer.validateRoot(msg.proof.merkleRoot):
+  #   debug "invalid message: provided root does not belong to acceptable window of roots", provided=msg.proof.merkleRoot, validRoots=rlnPeer.validMerkleRoots
+  #   waku_rln_invalid_messages_total.inc(labelValues=["invalid_root"])
+  #   return MessageValidationResult.Invalid
```

Comment:
This is only part that should be commented out for now, ideally with link to issue to fix

Reply:
done too https://github.com/status-im/nwaku/pull/1246/commits/a043a2ec5125ce7b789d25a8e187775752953710